server/sonar-db-core/src/main/resources/org/sonar/db/version/schema-h2.ddl

```diff
@@ -457,7 +459,9 @@ CREATE TABLE "USERS" (
   "USER_LOCAL" BOOLEAN,
   "ONBOARDED" BOOLEAN NOT NULL,
   "CREATED_AT" BIGINT,
-  "UPDATED_AT" BIGINT
+  "UPDATED_AT" BIGINT,
+  "HOMEPAGE_TYPE" VARCHAR(15),
```

Comment:
I suggest to have 40 like HOMEPAGE_VALUE, but let's discuss

Reply:
that the value i originally provided :/

Reply:
-> 40